### ISSUE TITLE
bug(Select): With VoiceOver + iOS, focus lands on the select label (not the combobox), so VO doesn't announce a dropdown/combobox

### ISSUE THREAD

**Issue by Joeh89** [angular/components#22524]
Reproduction
URL: https://material.angular.io/components/select/overview
Version: 11.2.9
All mat-select dropdowns.

Steps to reproduce:

Open VoiceOver on an iOS device and swipe to the first mat-select dropdown.
VO should read: Dropdown label + combobox (+ selected value)

Actual Behavior
a) On step 1: VO only reads Dropdown label. It doesn't alert the user that it can be activated/is a dropdown
b) If float-label is set to never, then VO skips entirely the dropdown, non-sighted users would never know that the dropdown exists.

Environment
Angular: Mat angular 11.2.9
CDK/Material: Mat angular 11.2.9
Browser(s): Safari
Operating System (e.g. Windows, macOS, Ubuntu): iOS

**Comment by zarend** [comment]
Filed this [internally](http://b/224983850)

**Comment by zarend** [comment]
Hmm, I cannot reproduce this. The "Basic mat-select" demo is announced as "combobox" for me on iOS VoiceOver. Maybe this already has been fixed.

But there are 2 navigation stops on the select. The first is for the "combobox" and the second is a label.

iOS 15.4.1
iPhone SE Model MP9D2LL/A

**Comment by Joeh89** [comment]
Yes, that's right, it's fixed. I believe the only issue remaining on iOS phone when VO is on is that the focus doesn't jump to the first option after expanding the drop-down. User is forced to explore by touch to manually place the focus on the dropdown container

**Comment by zarend** [comment]
Thanks, glad it's working better than before. iOS VO focus not jumping to the first option is being tracking in issue #24735, which has a PR in progress to fix.

I'll go ahead and close this ticket since the select is being properly announced as a combobox that can be interacted with instead of a label.
